spec/lib/epi_deploy/release_spec.rb

```diff
@@ -8,14 +8,15 @@ def initialize(options = {})
     @pending_changes = options[:pending_changes].nil? ? false : options[:pending_changes]
   end
   def add(files); end
-  def on_master?; @on_master; end
+  def on_master_or_main?; @on_master; end
   def pending_changes?; @pending_changes; end
   def short_commit_hash; 'abc1234'; end
   def commit(msg); end
   def tag(name); end
   def push(opts = {}); end
   def pull; end
   def change_branch_commit(branch, commit); end
+  def current_branch; 'master'; end
```

Comment:
If we're going to be wanting to move towards using `main` instead, the tests should probably signify that change.

Reply:
I agree - definitely needs test coverage for the `main` branch too

Reply:
I've changed the current branch in that test to main, and also added some tests for the git wrapper that test that both main and master are 'primary' branches.